tests/scanners/test.javascript.js

```diff
@@ -29,6 +29,16 @@ describe('JavaScript Scanner', function() {
     assert.equal(jsScannerWithOptions.options.foo, 'bar');
   });
 
+  it('should not throw when async/await is used', () => {
+    var code = singleLineString`var foo = async a => a;`;
```

Comment:
Because the code is not over 80 lines, so you don't need to use `singleLineString` here, and regular single quotes (`'`) should be fine. 😄